src/content/release-notes.mdx

```diff
@@ -5,6 +5,27 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.35.0
+
+1 August 2025
+
+This version is compatible with Kubernetes versions 1.31 to 1.33
+Okteto Chart release 1.35 is designed to work with [Okteto CLI 3.10.x](https://github.com/okteto/okteto/releases/tag/3.10.0)
+
+### New Features {#new-features-1.35}
+- CIDR-based traffic filtering is now supported for BYOC (Bring Your Own Cluster) environments. Use this to improve security by restricting access to Okteto services to specific IP ranges.
+- Added support for [Kubernetes 1.33](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.33.md) (support for 1.30 has been removed) and [Amazon Linux 2023](https://github.com/amazonlinux/amazon-linux-2023)
+- Namespace deletion now includes a configurable timeout to help with stuck cleanups  
```

Comment:
Namespace deletion always had a timeout flag to specify, but it wasn't propagated to dev environments with graceful destroy (the ones with a `destroy` section in the manifest). So, if the dev environment destruction was taking more than 5 minutes, the destroy namespace operation was failing. I think it might be relevant to give those details